File: SDKs/Aspose.Tasks_Cloud_SDK_for_CSharp/src/Com/Aspose/Tasks/Model/CalendarExceptionsResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class CalendarExceptionsResponse {
    public List<CalendarException> CalendarExceptions { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class CalendarExceptionsResponse {\n");
      sb.Append("  CalendarExceptions: ").Append(CalendarExceptions).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Tasks-Cloud-SDK-for-Java/src/main/java/com/aspose/tasks/model/CalendarExceptionsResponse.java
package com.aspose.tasks.model;

import java.util.*;
public class CalendarExceptionsResponse {
  private List<CalendarException> CalendarExceptions = new ArrayList<CalendarException>();
  private String Code = null;
  private String Status = null;
  /**
	 * getCalendarExceptions
	 * Gets List<CalendarException>
	 * @return CalendarExceptions
	 */
  public List<CalendarException> getCalendarExceptions() {
    return CalendarExceptions;
  }

	/**
	 * setCalendarExceptions
	 * Sets List<CalendarException>
	 * @param CalendarExceptions List<CalendarException>
	 */
  public void setCalendarExceptions(List<CalendarException> CalendarExceptions) {
    this.CalendarExceptions = CalendarExceptions;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class CalendarExceptionsResponse {\n");
    sb.append("  CalendarExceptions: ").append(CalendarExceptions).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

